Stripe/STPBankSelectionViewController.m

```diff
@@ -0,0 +1,136 @@
+#import "STPPaymentMethodParams.h"
+#import "STPPaymentMethodFPXParams.h"
+#import "UIBarButtonItem+Stripe.h"
+#import "UINavigationBar+Stripe_Theme.h"
+#import "UITableViewCell+Stripe_Borders.h"
+#import "UIViewController+Stripe_NavigationItemProxy.h"
+
+static NSString *const STPBankSelectionCellReuseIdentifier = @"STPBankSelectionCellReuseIdentifier";
+
+@interface STPBankSelectionViewController () <UITableViewDataSource, UITableViewDelegate>
+@property (nonatomic) STPAPIClient *apiClient;
+@property (nonatomic) STPBankType bankType;
+@property (nonatomic) STPFPXBankBrand selectedBank;
+@property (nonatomic) STPPaymentConfiguration *configuration;
+@property (nonatomic, weak) UIImageView *imageView;
+@property (nonatomic) STPSectionHeaderView *headerView;
+@property (nonatomic) BOOL loading;
+@end
+
+@implementation STPBankSelectionViewController
+
+- (instancetype)initWithBankType:(STPBankType)bankType {
+    return [self initWithBankType:bankType configuration:[STPPaymentConfiguration sharedConfiguration] theme:[STPTheme defaultTheme]];
+}
+
+- (instancetype)initWithBankType:(STPBankType)bankType
+                   configuration:(STPPaymentConfiguration *)configuration
+                           theme:(STPTheme *)theme {
+    self = [super initWithTheme:theme];
+    if (self) {
+        NSCAssert(bankType == STPBankTypeFPX, @"STPBankSelectionViewController currently only supports FPX.");
+        _bankType = bankType;
+        _configuration = configuration;
+        _selectedBank = STPFPXBankBrandUnknown;
+        _apiClient = [[STPAPIClient alloc] initWithConfiguration:configuration];
+        self.title = STPLocalizedString(@"Bank Account", @"Title for bank account selector");
+    }
+    return self;
+}
+
+- (void)createAndSetupViews {
+    [super createAndSetupViews];
+
+    [self.tableView registerClass:[STPBankSelectionTableViewCell class] forCellReuseIdentifier:STPBankSelectionCellReuseIdentifier];
+
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+}
+
+- (void)updateAppearance {
+    [super updateAppearance];
+    
+    self.tableView.allowsSelection = YES;
+    
+    [self.tableView reloadData];
+}
+
+- (BOOL)useSystemBackButton {
+    return YES;
+}
+
+#pragma mark - UITableView
+
+- (NSInteger)numberOfSectionsInTableView:(__unused UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(__unused UITableView *)tableView numberOfRowsInSection:(__unused NSInteger)section {
+    return STPFPXBankBrandUnknown;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView
+         cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    STPBankSelectionTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:STPBankSelectionCellReuseIdentifier forIndexPath:indexPath];
+    STPFPXBankBrand bankBrand = indexPath.row;
+    BOOL selected = self.selectedBank == bankBrand;
+    [cell configureWithBank:bankBrand theme:self.theme selected:selected enabled:!self.loading];
```

Comment:
Huh, every cell is configured with the same loading..I'd guess once `loading == YES` and you scroll the tableview, the cells that appear all have spinners 😃

Reply:
It's enabled (vs greyed out and disabled) based on `!self.loading`, it's selected (which shows the spinner) based on `selected`, which is `self.selectedBank == bankBrand`.

Reply:
Ah my mistake